pkg/server/container_create.go

```diff
@@ -213,7 +216,13 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 
 		// TODO(random-liu): [P1] Set selinux options.
 
-		// TODO(random-liu): [P2] Add apparmor and seccomp.
+		// TODO(random-liu): [P2] Add seccomp.
+
+		// Set apparmor profile name, Note: trims default profile name prefix
+		appArmorProf := securityContext.GetApparmorProfile()
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto#L512-L517

There are 3 possibilities:
1) runtime/default
2) localhost/<profile_name>
3) empty

We should also handle 1). I'm fine with making the default as empty for now, but we should add a TODO to make the default apparmor profile configurable via a flag or config file. And if the profile is `runtime/default`, you may not want to write it into the spec.